.github/workflows/sonarcloud.yml

```diff
@@ -0,0 +1,65 @@
+name: SonarCloud
+
+on:
+  push:
+    branches:
+      - "*"
+  # https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull_request_target
+  # We use this event so workflow can be run from the forked repo
+  pull_request_target:
+    types:
+      [
```

Comment:
I think these are far too many types. E.g. we don't need to rerun the analysis if a label gets changed or a review is requested. For our purposes `opened, edited, reopened, synchronize, unlocked` should be enough.

Reply:
Thanks, forgot to change that after all the testing ^^